modules/scripts/genindex.py

```diff
@@ -96,14 +96,23 @@ def get_index(root, subprojects, sub_docpaths, javadoc):
     # add javadoc if javadoc exist
     if javadoc != "":
         index = add_javadoc(index)
-
+    if build_dir != "":
+        index = add_subproject(index, build_dir)
     return '\n\n'.join(index) + '\n'
 
 def add_javadoc(index):
     javadoc = ["API Documentation", "\n", "=================", "\n",
      "* `javadoc <./javadoc>`_"]
     return index + javadoc
 
+def add_subproject(index, build_dir):
```

Comment:
@pri22296 now the documentation is not generated as a single project like before. now documentation is generated separately according to `.yaydoc.yaml` specified in each project. so I added a function which walks the folder and links it to the index.rst.